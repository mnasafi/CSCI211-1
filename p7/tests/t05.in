echo t05: Insert days of week into tree, then print size.
insert Saturday
insert Friday
insert Tuesday
insert Monday
insert Thursday
insert Wednesday
insert Sunday
echo This size of the tree is:
size
echo Should have printed 7 on the previous line.
