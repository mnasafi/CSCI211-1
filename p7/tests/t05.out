t05: Insert days of week into tree, then print size.
This size of the tree is:
7
Should have printed 7 on the previous line.
